infrastructure/template.yaml

```diff
@@ -22,6 +22,10 @@ Parameters:
     Type: String
     Default: common-cri-api
     Description: The name of the stack containing the common CRI lambdas/infra
+  CommonInfrastructureStackName:
+    Type: String
+    Default: txma-infrastructure
```

Comment:
common infra isn't just txma, it's just the repo also contains txma stack. Could we rename this parameter to TXMAStackName or similar please? Thanks

Reply:
Good point - done!